consensus/hotstuff/verification/combined_signer_v2_test.go

```diff
@@ -124,7 +160,7 @@ func TestCombinedSignWithNoDKGKey(t *testing.T) {
 	committee.On("DKG", mock.Anything).Return(dkg, nil)
 
 	packer := signature.NewConsensusSigDataPacker(committee)
-	verifier := NewCombinedVerifierV2(committee, packer)
+	verifier := NewCombinedVerifier(committee, packer)
```

Comment:
may I request to correct the godoc below: 
https://github.com/onflow/flow-go/blob/8778e8a90c5ade7b9b30afcf123f65ae769a5050/consensus/hotstuff/verification/combined_signer_v2_test.go#L173
Here, we are explicitly testing the case, where the signer does _not_ have a random beacon key. Therefore, I think the comment should be corrected.

I would suggest to replace: 
https://github.com/onflow/flow-go/blob/8778e8a90c5ade7b9b30afcf123f65ae769a5050/consensus/hotstuff/verification/combined_signer_v2_test.go#L173-L179
by 
```golang
// As the proposer does not have a Random Beacon Key, it should sign soley with its staking key.
// In this case, the SigData should be identical to the staking sig.
expectedStakingSig, err := stakingPriv.Sign(
     MakeVoteMessage(block.View, block.BlockID),
     crypto.NewBLSKMAC(encoding.ConsensusVoteTag),
)
require.NoError(t, err)
require.Equal(t, expectedStakingSig, crypto.Signature(proposal.SigData))
```